source/_outdated_notice.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="refills-wrapper">
+  <div class="refills-notice">
+    <span>
+      Refills requires Bourbon 5.0+ and Neat < 2.0, >= 1.6. These have not been
```

Comment:
I think we can cut `These have not been updated to support the most recent versions.`.